stable/coredns/values.yaml

```diff
@@ -275,6 +287,15 @@ autoscaler:
     # i.e. strategy.spinnaker.io/versioned: "false" to ensure configmap isn't renamed
     annotations: {}
 
+  # Enables the livenessProbe for cluster-proportional-autoscaler - this requires version 1.8.0+ of the autoscaler
```

Comment:
this probably should not be in here as it was merged in a separate PR

Reply:
Closed and replaced by https://github.com/coredns/helm/pull/36